src/test/java/org/example/SearchFieldTest.java

```diff
@@ -0,0 +1,62 @@
+package org.example;
+
+import org.example.page.MainPage;
+import org.example.page.SearchResultPage;
+import org.junit.*;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.concurrent.TimeUnit;
+
+public class SearchFieldTest {
+    private WebDriver driver;
+
+    private static final String INCORRECT_INFORMATION = "guihjkbgrsxcfhhbpppppp";
+    private static final String NO_RESULTS_FOR = "No results for ";
+    private static final String LAPTOP = "laptop";
+    private static final String EXPECTED_HEADER = "\"" + LAPTOP + "\"";
+
+
+    @BeforeClass
+    public static void setUpClass() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/webdriver/chromedriver.exe");
+    }
+
+    @Before
+    public void setUpTest() {
+        driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+    }
+
+    @Test
+    public void incorrectInput() {
+        MainPage mainPage = new MainPage(driver);
+        mainPage.openPage().searchItem(INCORRECT_INFORMATION);
+        String wr = new SearchResultPage(driver).getIncorrectSearchResultText();
```

Comment:
You don't need to create a new instance of SearchResultPage, because you return this instance in your MainPage class method 
public SearchResultPage searchItem(String item) {

So you can easily write like this 
`String wr = mainPage.openPage().searchItem(INCORRECT_INFORMATION).getIncorrectSearchResultText();`

Reply:
and it is better to rename this wariable wr to something more understandable, e.g. "incorrectSearchValue"